module_utils/ansible_nailgun_cement.py

```diff
@@ -196,9 +246,18 @@ def parse_template_from_file(file_name, module):
 
 
 def find_organization(module, name, failsafe=False):
-    org = Organization(name=name)
-    response = org.search(set(), {'search': 'name="{}"'.format(name)})
-    return handle_find_response(module, response, message="No organization found for %s" % name, failsafe=failsafe)
+    org = Organization(name=name).search(set(), {'search': 'name="{}"'.format(name)})
+    return handle_find_response(module, org, message="No organization found for %s" % name, failsafe=failsafe)
+
+
+def find_location(module, name, failsafe=False):
+    loc = Location(name=name).search(set(), {'search': 'name="{}"'.format(name)})
+    return handle_find_response(module, loc, message="No location found for %s" % name, failsafe=failsafe)
+
+
+def find_compute_resource(module, name, failsafe=False):
+    cr = AbstractComputeResource(name=name).search(set(), {'search': 'name="{}"'.format(name)})
```

Comment:
I'd prefer if the variable would have more than 2 characters, but given we only ever use it once in the next line this is not a hard req ;)